app/src/Components/Welcome/WelcomeButton.tsx

```diff
@@ -1,15 +1,23 @@
-//'Get Started' button allows user to go to main part of app from any slide
-
+/**
+ * @file 'Get Started' button allows user to go to main part of app from any slide
+ */
 import React from 'react'
-import {TouchableOpacity, Text} from 'react-native'
+// import { TouchableOpacity, Text } from 'react-native'
```

Comment:
Please can you remove this line if no longer needed, rather than commenting it out

Reply:
I'm not sure what this is actually, both were flagged in the linting checks for going through unused, but I'll remove them for now.

Reply:
This line has been removed